core/src/main/java/org/apache/iceberg/rest/RESTSessionCatalog.java

```
@@ -462,6 +463,26 @@ private LoadTableResponse loadInternal(
 
   @Override
   public Table loadTable(SessionContext context, TableIdentifier identifier) {
+    try {
+      return loadTableInternal(context, identifier);
+    } catch (NoSuchTableException original) {
+      return loadViewMetadataTable(context, identifier);
+    }
+  }
+
+  private Table loadViewMetadataTable(SessionContext context, TableIdentifier identifier) {
+    String tableName = identifier.name();
+    ViewMetadataTableType type = ViewMetadataTableType.from(tableName);
+    if (type != null) {
+      TableIdentifier baseViewIdentifier = TableIdentifier.of(identifier.namespace().levels());
+      View baseView = loadView(context, baseViewIdentifier);
+      return MetadataTableUtils.createViewMetadataTableInstance(baseView, tableName, type);
+    } else {
+      throw new NoSuchTableException("Table does not exist: %s", identifier);
```

we should probably throw the original exception that was caught in `loadTable`